### ISSUE TITLE
Adding Accessibility to Linux Electron Apps

### ISSUE THREAD

**Issue by ghost** [electron/electron#11544]
* Electron version: various
* Operating system: Arch Linux

### Expected behavior
There should be a way to put the ChromeVox Chrome/Chromium addon in to an electron app, via something like --inject-addon or putting it in to a directory that all Electron apps source on start. This is necessary so that blind people can take advantage of Electron apps; as it stands now, neither Chromium or Chrome work with the Orca screenreader, and the browser requires ChromeVox to serve as a screenreader.

### Actual behavior
I can't find any such way to carry this out. I looked in /usr/lib and /lib, but I don't see any of the apps in there; with that said, I also didn't find an --inject-addon or similar command line switch anywhere, either.

### How to reproduce
I tried this with the Spotify and visual-studio-code-bin packages.

**Comment by akashnimare** [comment]
@sofianguy have a look https://github.com/signalapp/Signal-Desktop/issues/2178#issuecomment-384911415.

**Comment by tcikoritys** [comment]
This, times 1000. More and more developers are turning to Electron to write their cross-platform things. Yes, they work on Windows, Mac, and Linux, but the accessibility accross all those systems varies from totally usable and accessible to completely useless and inaccessible. The worst of it is that some apps that were accessible are no longer. Skype's QT app, while not perfect, was at least workable on Linux with Orca. Now Orca can see nothing. No controls, no text.

I'm sure I don't need to mention all the apps people make with Electron -- everything from blockchain wallet management apps and dapps, to IM solutions (Skype, Signal), to IDEs and editors (Atom, VSCode)... And the list is growing every day.

I love the idea that Electron brings to the table. But with no way to make the apps accessible on Linux, the more apps that are written in it, the less the modern Linux desktop is usable for blind people.

It may be that Electron itself cannot solve this issue directly, though I'm not sure how that would be right now, as it seems there are ways of adding some Chrome extensions at build time. Speaking on behalf of the community, however, this does need to be addressed and brought to the appropriate people, and any feedback your team can provide to help us realize this goal would be invaluable, as this problem is only growing. Again, I have no problem with the advantages Electron brings. I love seeing people able to offer their apps in a cross-platform way. But as things are now, every new cross-platform app that is made or remade in Electron is a loss to the blind/visually impaired/disabled/... Linux community, which only serves to harm the ecosystem as a whole. And this, I'm sure, no one would want or do intentionally.

**Comment by rameshc-clarion** [comment]
I am also looking to add chrome vox addon to add in chromium or Electron .
Does any one knows the way to add chrome vox to electron ? 
guide me

**Comment by isidorn** [comment]
fyi linux orca works fine with VS Code Insiders which is using Electron 7.
More details here https://github.com/microsoft/vscode/issues/19522#issuecomment-588145223

**Comment by codebytere** [comment]
@tcikoritys this is no longer the case, as stated by @isidorn above. Chromium now has [support](https://wiki.gnome.org/Projects/Orca/Chromium) for Orca, which is available to Electron developers and users via the ` --force-renderer-accessibility` flag. 

This is the best path forward for this issue now, as taking advantage of Chromium offerings is significantly better for maintainability and support than trying to jack in random addons. We care about and will continue to work to provide the most accessible experience for users to the best of our ability, so we welcome any further feedback that you all may have.
